examples/mi-mctp.c

```diff
@@ -184,24 +184,24 @@ int do_identify(nvme_mi_ep_t ep, int argc, char **argv)
 	uint16_t ctrl_id;
 	char buf[41];
 	bool partial;
-	int rc, tmp;
+	int rc;
 
 	if (argc < 2) {
 		fprintf(stderr, "no controller ID specified\n");
 		return -1;
 	}
 
-	tmp = atoi(argv[1]);
-	if (tmp < 0 || tmp > 0xffff) {
+	ctrl_id = atoi(argv[1]);
+	if (ctrl_id < 0 || ctrl_id > 0xffff) {
```

Comment:
These checks are no longer possible if you're using `ctrl_id` directly there, as it's a `uint16_t`. Values outside of that 16-bit unsigned range will get silently dropped, instead of the check failing.

Reply:
yup, done